sklearn/utils/tests/test_array_api.py

```diff
@@ -62,18 +62,21 @@ def test_get_namespace_ndarray_creation_device():
 def test_get_namespace_ndarray_with_dispatch():
```

Comment:
@ogrisel @lesteve this is the main change to be compatible with the new `array-api-compat` release.